faker/providers/file/__init__.py

```diff
@@ -204,6 +204,18 @@ class Provider(BaseProvider):
             ("video", video_file_extensions),
         )
     )
+
+    file_systems_path_rules: Dict[str, Dict] = {
+        "windows": {
+            "root": "C:\\",
+            "seperator": "\\",
+        },
+        "linux": {
+            "root": "/",
+            "seperator": "/",
+        },
```

Comment:
The correct spelling is `separator`:

```suggestion
        "windows": {
            "root": "C:\\",
            "separator": "\\",
        },
        "linux": {
            "root": "/",
            "separator": "/",
        },
```